packages/fx-core/src/plugins/resource/bot/errors.ts

```diff
@@ -104,18 +105,33 @@ export class PluginError extends Error {
 
   genMessage(): string {
     let msg = `${this.details[0]} `;
+    msg += this.getAppStudioErrorDetails(this.innerError);
     if (this.suggestions.length > 0) {
       msg += getDefaultString("plugins.bot.ErrorSuggestions", this.suggestions.join(" "));
     }
     return msg;
   }
+
   genDisplayMessage(): string {
     let msg = `${this.details[1]} `;
+    msg += this.getAppStudioErrorDetails(this.innerError);
```

Comment:
Why don't we also output inner error besides AppStudio Error?

Reply:
We can have a better structure of the error message. E.g.
message: ..., suggestion: ..., innerError: ...

Reply:
The `getAppStudioErrorDetails` actually returns the inner error message, I will change it to a better name.